module-presentation/src/main/kotlin/site/yourevents/health/api/HealthController.kt

```diff
@@ -0,0 +1,14 @@
+package site.yourevents.health.api
+
+
+import org.springframework.web.bind.annotation.RestController
+import site.yourevents.response.ApiResponse
+import site.yourevents.type.SuccessCode
+
+@RestController
+class HealthController : HealthApi {
+
+    override fun healthCheck(): ApiResponse<String> {
+        return ApiResponse.success(SuccessCode.REQUEST_OK, "ok")
```

Comment:
마찬가지로 다음과 같이 변경해주시면 좋을 것 같습니다!
```suggestion
    override fun healthCheck(): ApiResponse<Unit> {
        return ApiResponse.success(SuccessCode.REQUEST_OK)
```